omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -214,4 +214,18 @@ public void searchByPatient_shouldGetUnretiredVisitsForThePatient() throws Excep
 		Assert.assertEquals(3, Util.getResultsSize(result));
 	}
 	
+	@Test
+	public void searchByPatient_shouldGetRetiredVisitsIfIncludeAllIsTrue() throws Exception {
+		String patient = "da7f524f-27ce-4bb2-86d6-6d1d05312bd5";
+		String visitUuid = "e1428fea-6b78-11e0-93c3-18a905e044dc";
+		handle(newDeleteRequest(getURI() + "/" + visitUuid, new Parameter("reason",
```

Comment:
Can you also show how many results are returned for both cases, before you do the newDeleteRequest?